1.Exploratory_Data_Analysis/scripts/2.extended_umap_analysis.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>Why 5000?</p><p> </p><p>May also want to change the <code>random_state</code> to <code>0</code> to align with way standards</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>5000 is arbitrary, and good catch.  I will change random state now</p>